packages/core/src/components/overlay/_overlay.scss

```diff
@@ -72,19 +72,14 @@ Styleguide components.overlay.scroll
 
 $overlay-background-color: rgba($black, 0.7);
 
-// scroll container for non-inline overlay
-.pt-overlay-scroll-container.pt-overlay-open {
-  @include position(absolute, 0);
-  overflow: auto;
-
-  .pt-overlay-backdrop {
-    // fixed position so it won't scroll under the overlay
-    position: fixed;
```

Comment:
Deleted this rule (and therefore this `.pt-overlay-backdrop` block), because the backdrop is set to `position: fixed` by default now.